src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { MAILER_API_URL } from '../config/config.example.json';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
```

Comment:
instead of a for loop consider doing [`array.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) so like

```js
files.map(file => {
  // the "file" variable is equivalent to files[i]
})
```